temporalcli/commandsmd/commands.md

```diff
@@ -338,7 +338,7 @@ Includes options set for [payload input](#options-set-for-payload-input).
 * `--run-id`, `-r` (string) - Run Id. Cannot be set when query is set.
 * `--query`, `-q` (string) - Start a batch to Signal Workflow Executions with given List Filter. Either this or
   Workflow Id must be set.
-* `--reason` (string) - Reason to perform batch. Only allowed if query is present unless the command specifies otherwise. 
+* `--reason` (string) - Reason to perform batch. Only allowed if query is present unless the command specifies otherwise. Defaults to message with the current user's name.
```

Comment:
This still mentions batch?

Reply:
You're looking at the help for `workflow signal` rather than `workflow terminate`. That command only allows a reason to be specified when invoked in batch mode. If you check the options for `workflow terminate` you'll see I updated it there already:

```
* `--reason` (string) - Reason to terminate this workflow. Defaults to message with the current user's name.
```